backend/scripts/test_table_creation_debug.py

```
@@ -0,0 +1,91 @@
+LLMレスポンスの詳細を確認
+"""
+
+import asyncio
+import logging
+import sys
+from pathlib import Path
+
+# プロジェクトルートをPythonパスに追加
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+from app.core.config import settings
```

Ruff (0.11.9)

`app.core.config.settings` imported but unused

Remove unused import: `app.core.config.settings`

(F401)